homeassistant/components/device_tracker/tado.py

```diff
@@ -0,0 +1,109 @@
+"""
+homeassistant.components.device_tracker.tado.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Device tracker platform that supports presence detection.
+The detection is based on geofencing enabled devices used with Tado.
+Tado is a Smart Thermostat device.
+"""
+import logging
+from datetime import timedelta
+from collections import namedtuple
+
+import requests
```

Comment:
`device_tracker` is now async. Please use aiohttp like upc does (or components/rest_commands.py).